cpu/saml21/periph/rtc.c

```diff
@@ -97,7 +97,7 @@ void rtc_init(void)
 
     /* RTC config with RTC_MODE2_CTRL_CLKREP = 0 (24h) */
     RTC->MODE2.CTRLA.reg = RTC_MODE2_CTRLA_PRESCALER_DIV1024 |  /* CLK_RTC_CNT = 1KHz / 1024 -> 1Hz */
-#if (SAML21XXXB) || (SAMR30)
+#if (CPU_MODEL_SAML21J18B)
```

Comment:
How about changing this like in this patch:

https://gist.github.com/9842253687bd2698a9b344779dd83500

E.g., define the bit if not visible, and always set it.
I can confirm that setting the bit on SAML21 rev A doesn't hurt.

Reply:
> I can confirm that setting the bit on SAML21 rev A doesn't hurt

Then I'll change it :)